code/modules/mob/living/carbon/xenomorph/Embryo.dm

```diff
@@ -98,6 +98,14 @@
 
 	switch(stage)
 		if(2)
+			if(prob(1))
+				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_CHESTBURST_WEAK)
+					affected_mob.visible_message(SPAN_DANGER("\The [affected_mob] starts shaking uncontrollably!"), \
```

Comment:
```suggestion
					affected_mob.visible_message(SPAN_DANGER("[affected_mob] starts shaking uncontrollably!"), \
```

Reply:
I agree. (This is just a holdover from the existing paralyze section anyways)